specification/containerservice/Fleet.Management/update/run.tsp

```diff
@@ -135,6 +136,9 @@ enum ManagedClusterUpgradeType {
 
   @doc("NodeImageOnly upgrades only the node images of the target ManagedClusters.")
   NodeImageOnly,
+
+  @doc("ControlPlaneOnly upgrades only the control plane of the target ManagedClusters.")
+  ControlPlaneOnly,
```

Comment:
Use decorator @added(Versions.v2024_02_02_preview) to avoid changing released versions